src/templates/blog-post.tsx

```diff
@@ -100,4 +165,24 @@ export const BlogPostPageQuery = graphql`
   }
 `;
 
+export type BlogArticleQueryData = ContentfulBlogPost & {
+  code: {
+    childMarkdownRemark: {
+      htmlAst: any;
+    };
+  };
+  readingTime: {
+    minutes: number;
+  };
+  heroImage: {
+    fullImage: IGatsbyImageData;
+    squaredImage: IGatsbyImageData;
```

Comment:
This is obsolete now, isn't it after the recent change of the blog hero with the new flag `naturalHeight` ?